src/main/java/org/cloudbees/literate/jenkins/publishers/Agent.java

```diff
@@ -76,7 +76,7 @@ protected Agent(Class<P> clazz) {
      */
     @CheckForNull
     public abstract P getPublisher(@NonNull BuildListener listener, @NonNull FilePath configurationFile)
-            throws IOException;
+            throws IOException, InterruptedException;
```

Comment:
I don't see who is throwing this exception on implementations, is it really required?